src/genericComponents/Checkbox.tsx

```diff
@@ -0,0 +1,21 @@
+import React, { useState } from 'react';
+import { InputGroup } from 'react-bootstrap';
+import styled from 'styled-components';
+
+const Checkbox = (props: any) => {
+  const [clicked, setClicked] = useState(false);
+
+  return <CAGCheckbox onClick={() => setClicked(!clicked)}></CAGCheckbox>;
```

Comment:
For now, let's just do:
```
return <CAGCheckbox {...props} />;
```

We can always get more specific later if we need to